reference/test/stop/residual_norm_kernels.cpp

```diff
@@ -93,14 +93,15 @@ TYPED_TEST(ResidualNorm, CheckIfResZeroConverges)
                              .on(this->exec_)
                              ->generate(mtx, rhs, x.get(), nullptr);
         constexpr gko::uint8 RelativeStoppingId{1};
-        bool one_changed{};
+        gko::array<bool> stop_indicators(this->exec_->get_master(), 2);
         gko::array<gko::stopping_status> stop_status(this->exec_, 1);
         stop_status.get_data()[0].reset();
 
         EXPECT_TRUE(criterion->update().residual_norm(res_norm).check(
-            RelativeStoppingId, true, &stop_status, &one_changed));
+            RelativeStoppingId, true, &stop_status,
```

Comment:
General thought: This boolean parameter doesn't tell us what it does. Maybe it could be an enum?